src/list.json

```diff
@@ -212,7 +212,16 @@
         "link": "https://github.com/pusher/hacktoberfest/blob/main/README.md"
       }
     ],
-    "Q": [],
+    "Q": [
+      {
+        "organization": "QuestDB",
+        "org_url": "https://questdb.io/",
+        "tags": ["Swag", "TShirt", "Sticker"],
```

Comment:
```suggestion
        "tags": ["Swag", "TShirt" ,"Sticker"], => "Tshirt"  
```

Reply:
I change the suggestion, please do check!